server/repository/src/db_diesel/sync_log.rs

```diff
@@ -0,0 +1,351 @@
+use super::{
+    sync_log_row::{sync_log, sync_log::dsl as sync_log_dsl},
+    StorageConnection,
+};
+
+use crate::{
+    diesel_macros::{
+        apply_date_time_filter, apply_equal_filter, apply_simple_string_filter, apply_sort,
+    },
+    DBType, DatetimeFilter, EqualFilter, Pagination, RepositoryError, SimpleStringFilter, Sort,
+    SyncLogRow,
+};
+
+use diesel::prelude::*;
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct SyncLog {
+    pub sync_log_row: SyncLogRow,
+}
+
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct SyncLogFilter {
+    pub id: Option<EqualFilter<String>>,
+    pub started_datetime: Option<DatetimeFilter>,
+    pub done_endtime: Option<DatetimeFilter>,
+    pub prepare_initial_start_datetime: Option<DatetimeFilter>,
+    pub prepare_initial_done_datetime: Option<DatetimeFilter>,
```

Comment:
Might not need all these filters 🤔

Reply:
Sorry, but can you please delete, only keep the ones that are used